assets/messages.yml

```diff
@@ -354,22 +369,6 @@ messages:
 
         If you can provide a list of steps that someone else can follow to consistently reproduce a crash then feel free to create a new report. Please be sure to always search for duplicates before creating any new report. Please see the [*Bug Tracker Guidelines*|https://aka.ms/MCBugTrackerHelp] for more information.
 
-        %quick_links%
-      fillname: []
-    - project:
-        - mcd
-        - mclg
-      name: Crash logged automatically
-      shortcut: autocrash
```

Comment:
How come this one is removed?

Reply:
MCD/MCLG are obsolete

Reply:
Oh.. right. From the diff it looked like it had been removed completely, not just the two projects